tests/modules/nf-core/smncopynumbercaller/main.nf

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { SMNCOPYNUMBERCALLER } from '../../../../modules/nf-core/smncopynumbercaller/main.nf' addParams( options: [args: ''] )
```

Comment:
Can you move these options to the `nextflow.config` file? Also to stay consistent with other modules